boot/zephyr/include/sysflash/sysflash.h

```diff
@@ -45,8 +53,10 @@
 
 #else /* CONFIG_SINGLE_APPLICATION_SLOT */
 
-#define FLASH_AREA_IMAGE_PRIMARY(x)	FIXED_PARTITION_ID(slot0_partition)
-#define FLASH_AREA_IMAGE_SECONDARY(x)	FIXED_PARTITION_ID(slot0_partition)
+#define FLASH_AREA_IMAGE_PRIMARY(x)                             \
+            (ZEPHYR_MCUBOOT_APP_0_PRIMARY_SLOT_ID + (x - x)) 
+#define FLASH_AREA_IMAGE_SECONDARY(x)                           \
+            (ZEPHYR_MCUBOOT_APP_0_PRIMARY_SLOT_ID + (x - x))
```

Comment:
Is x needed here at all?
i.e.
```suggestion
            (ZEPHYR_MCUBOOT_APP_0_PRIMARY_SLOT_ID) 
#define FLASH_AREA_IMAGE_SECONDARY(x)                           \
            (ZEPHYR_MCUBOOT_APP_0_PRIMARY_SLOT_ID)
```

Reply:
This is to prevent variable passed as `x` to be detected as "unused".